app/screens/InfoScreen/SponsorCard.tsx

```diff
@@ -29,6 +29,34 @@ export type TierLevelProps =
 
 export type SponsorCardProps = CommonProps & TierLevelProps
 
+function maxImageDimensions(tier: "Platinum" | "Gold" | "Silver" | "Bronze"): {
```

Comment:
I would like to extract the types from the type information above so that I'm not duplicating it here.

Reply:
♻️ Completed.